### ISSUE TITLE
Propensity Scores

### ISSUE THREAD

**Issue by statasaurus** [PSIAIMS/CAMIS#45]

**Comment by andreaczhang** [comment]
I will try to do the R part for this topic!

**Comment by andreaczhang** [comment]
I'm not entirely sure about how to use propensity score as a multiple imputation technique; I would be glad to leave this topic to someone who knows!

**Comment by DrLynTaylor** [comment]
I dont think we meant this to be Multiple imputation.. it was just Propensity scores!   I have someone wanting to write in this area, so i'll bring them on board & assign them to this topic to hopefully move this one forward.

**Comment by DrLynTaylor** [comment]
Just noticed we have Issue #66 so can close this one and leave the other one
